third-party/imgui/imgui_impl_glfw.cpp

```diff
@@ -197,8 +197,12 @@ void ImGui_ImplGlfw_CharCallback(GLFWwindow* window, unsigned int c)
     io.AddInputCharacter(c);
 }
 
-void ImGui_ImplGlfw_MonitorCallback(GLFWmonitor*, int)
+void ImGui_ImplGlfw_MonitorCallback(GLFWmonitor* monitor, int event)
 {
+  ImGui_ImplGlfw_Data* bd = ImGui_ImplGlfw_GetBackendData();
+  if (bd->PrevUserCallbackMonitor != NULL)
+    bd->PrevUserCallbackMonitor(monitor, event);
+
```

Comment:
wow, was imgui just missing this originally?

Reply:
Yep, looking at the current release you can find the same empty function here https://github.com/ocornut/imgui/blob/v1.88/backends/imgui_impl_glfw.cpp#L397. Absolutely no idea why it's like this.